src/lib/features/feature-toggle/feature-collaborators-read-model.ts

```diff
@@ -31,6 +31,8 @@ export class FeatureCollaboratorsReadModel
             .join('users', 'recent_events.created_by_user_id', 'users.id')
             .orderBy('recent_events.max_created_at', 'desc');
 
+        console.log(query.toQuery());
+
```

Comment:
You probably wanna get rid of this.

```suggestion
```

Reply:
Ah, you got there before me 😄

Reply:
already did as you were writing the comment :D